lib/msprime.c

```diff
@@ -5954,6 +5949,34 @@ msp_multi_merger_common_ancestor_event(msp_t *self, double x,
     return ret;
 }
 
+/* Rejection method. */
```

Comment:
Looks good @grahamgower. The sake of legibility, I think it'd be good to put these functions towards the top of the file with some other general utilities, and to add a quick docstring-like comment about what each function does.